crypto/evp/e_des3.c

```diff
@@ -7,6 +7,12 @@
  * https://www.openssl.org/source/license.html
  */
 
+/*
+ * SHA-1 low level APIs are deprecated for public use, but still ok for
+ * internal use.
+ */
```

Comment:
Convert to EVP?

Reply:
Wait, what? This is an EVP backend...

Reply:
Yeah, there is some really crazy stuff in the code if you dig much beyond the superficial.

I was intending to convert this and the others to EVP next week.  Unless a good reason is given for not doing so.  Using EVP will add the cost of a level of function pointer indirection but could gain HW acceleration benefits.

Unless I'm missing something -- which seems to be quite common recently :(

Reply:
Removed this.

Reply:
> Wait, what? This is an EVP backend...

Its an EVP backend for DES not SHA1. It seems reasonable to still use EVP for the SHA-1 bits.

Reply:
> Its an EVP backend for DES not SHA1

Goo point.  Okie